corefxlab.sln

```diff
@@ -73,6 +73,10 @@ Project("{8BB2217D-0F2D-49D1-97BC-3654ED321F3B}") = "System.Time.Tests", "tests\
 EndProject
 Project("{8BB2217D-0F2D-49D1-97BC-3654ED321F3B}") = "System.Reflection.Metadata.Cil", "src\System.Reflection.Metadata.Cil\System.Reflection.Metadata.Cil.xproj", "{12D7B81D-26FD-4BCA-9D72-AFDEA900BAAC}"
 EndProject
+Project("{8BB2217D-0F2D-49D1-97BC-3654ED321F3B}") = "System.Text.Primitive", "src\System.Text.Primitive\System.Text.Primitive.xproj", "{80E6332B-DC83-4DFC-9C19-8D89E988BA4E}"
```

Comment:
We usually use plural nouns for namespace names. Could you rename it to System.Text.Primitives?